docs/source/api.rst

```diff
@@ -825,7 +850,7 @@ Returning a live result object would prevent the driver from correctly managing
 
 This function will receive a :class:`neo4j.ManagedTransaction` object as its first parameter.
 
-.. autoclass:: neo4j.ManagedTransaction
+.. autoclass:: neo4j.ManagedTransaction()
```

Comment:
why the parentheses here? `GraphDatabase` does not have them

Reply:
That tells the docs tool (sphinx) to not include the signature of the class. That's because the user is not supposed to instantiate this class themselves. Therefore, I don't want to document the constructor.